frontend_v2/src/app/components/challenge/challengeviewallsubmissions/challengeviewallsubmissions.component.html

```diff
@@ -228,6 +209,26 @@ <h5 class="fw-light">All Submissions</h5>
                     </strong>
                   </p>
                 </div>
+                <div class="element">
+                  <p>
+                    <strong class="text-light-black fw-semibold fs-12"
+                      >Metadata file&nbsp;<i class="fa fa-file fs-12" aria-hidden="true"></i></strong
+                    >&nbsp;
+                    <a
+                      target="_blank"
+                      class="blue-text fs-12"
+                      [href]="element.submission_metadata_file"
+                      *ngIf="element.submission_metadata_file"
+                    >
+                      <i class="fa fa-external-link"></i> Link
+                    </a>
```

Comment:
@savish28 we want all the options to be vertically aligned in a column. Can you please make that change too. Also, you can refer the `My submissions` page alignment for this.

Reply:
@Ram81 By options which one of the following arrangement you mean
- Vertically aligning all the three columns i.e with only one detail, status and action per row.
or
- Vertically aligning only the actions `leaderboard, Flagged, Re-run submission, Delete submission`

Reply:
@savish28 we should vertically align all fields for all 3 columns